### DISCUSSION TITLE
Turn off song names

### DISCUSSION THREAD

**Discussion by webma5ter23** [AzuraCast/AzuraCast#3995] [Q&A]
AzuraCast • v0.11.2 Stable • Docker • Ubuntu 20.04.1 LTS •  PHP 7.4

Hello, it's just a question. When it is transmitted live, no metadata is sent to the streaming, for this reason the last song transmitted by the autodj is fixed in the player.

I want the name of the radio to appear in the player and application while it is broadcasting live.

The software used is GRadio and Butt

I hope you can help me, thank you.

**Comment by pixel-paul** [comment]
You can send the meta data yourself using butt:

https://danielnoethen.de/butt/manual.html#_streaming

**Comment by webma5ter23** [comment]
I know but my question is not that, butt you need a text file and GRadio does not provide it that is why I want to disable everything from the panel and only show the name of the radio or only when it is broadcast live.

**Comment by silvaalex2021** [comment]
I think I know what you mean. Here in Brazil I used to know of a panel called voxtream, which had an option on the settings dialog to fill in the information you wanted to appear to listeners when broadcasting live, and when no metadata was provided.

I think that in azuracast there are no ways to do that today. Maybe you should make a suggestion to the developers to implement something similar via Featurebot, on the official link: https://features.azuracast.com/
